package.json

```diff
@@ -33,7 +33,10 @@
   },
   "homepage": "https://github.com/fastpack/fastpack#readme",
   "resolutions": {
-    "@esy-ocaml/reason": "github:facebook/reason#335c3fb2470964c2b4a7d571a8fbd9acdc3c2323"
+    "@esy-ocaml/reason": "github:facebook/reason#335c3fb2470964c2b4a7d571a8fbd9acdc3c2323",
+    "@opam/ssl": "github:bryphe/ocaml-ssl#8a4ac6a",
+    "@opam/ocplib-endian": "github:bryphe/ocplib-endian#d89d67d",
```

Comment:
This change is temporary until this override makes it in: https://github.com/esy-ocaml/esy-opam-override/pull/29